client/src/components/PhotosModal.jsx

```diff
@@ -54,15 +94,19 @@ const PhotosModal = (props) => {
   let [openModal, setModal] = useState(false);
   return (
     <PhotosModalWrapper>
-      <X>x</X>
+      <X onClick={() => props.toggleModal()}>x</X>
```

Comment:
Maybe you could  change the name of this styled component from X, or maybe you have a good reason for naming it "X".

Reply:
Yeah, maybe I could. It's literally just an x to close the modal so I thought it would be appropriate. I'm sure I could write something that better conveys what it does.